include/simdjson/nonstd/string_view.hpp

```diff
@@ -0,0 +1,1529 @@
+// Copyright 2017-2019 by Martin Moene
+//
+// string-view lite, a C++17-like string_view for C++98 and later.
+// For more information see https://github.com/martinmoene/string-view-lite
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#ifndef NONSTD_SV_LITE_H_INCLUDED
```

Comment:
Am I correct in thinking that this guard here on the NONSTD_SV_LITE_H_INCLUDED macro would make it so that if someone is already using string-view lite, this content would not be included? Presumably, this applies to our single-header version as well, right?

Reply:
Good point! That makes things somewhat better. It'll solve the problem if the user is using a binary-compatible version of string_view (even if it's a different version). If not, it's possible for simdjson.cpp to use one version, and their .cpp to use another.